app/assets/stylesheets/application.scss

```diff
@@ -68,6 +120,7 @@ body {
 }
 
 .footer {
+  font-family: $font-stack;
```

Comment:
Properties should be ordered background-color, font-family, font-size, padding, text-align